vms/setup.cfg

```diff
@@ -1,5 +1,15 @@
-# PEP 8 BEHAVIOUR
+[flake8]
+ignore = F841, W391
+exclude =
+    # Ignore git directories
+    ../.git,
+    # Ignore docs and automated docs
+    ../aut_docs/*, ../docs/*,
+    # Ignore manage.py
+    manage.py,
+    # Ignore migration files
+    *migrations*
+    # Ignore wsgi and
+    vms/wsgi.py
 
-[pep8]
-ignore = E123, E127, E128, E303, E713, E111, E241, E302, E121, E261, W391
-max - line - length = 160
+max-line-length = 120
```

Comment:
we are changing this right?

Reply:
Yeah I have it on my todo for afternoon.

Reply:
Done.